Source/Fluxor/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Fluxor.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace Fluxor.Extensions
+{
+	public static class IServiceCollectionExtensions
```

Comment:
Does everything still work okay if this is an internal class?

Reply:
no - it's used by ReduxDevTools project.

if you want it internal, then we could 'link' the .cs file in both projects, but that gets messy and is not always intuitive